geopandas/plotting.py

```diff
@@ -414,6 +414,11 @@ def plot_series(
         )
         return ax
 
+    # have colors been given for all geometries?
+    color_given = isinstance(color, (list, pd.core.series.Series, np.ndarray)) and len(
```

Comment:
```suggestion
    color_given = pd.api.types.is_list_like(color) and len(
```

this is a better check

Reply:
I googled for something like that - I thought there must be a utility function in pandas :-)